cli/azd/pkg/auth/testdata/certificate.pem

```diff
@@ -0,0 +1,49 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
I took this file from the testdata folder of azure-sdk-for-go, so no concern about checking it in.